<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>zuluCrypt</class>
 <widget class="QMainWindow" name="zuluCrypt">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>659</width>
    <height>375</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>zuluCrypt</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>zuluCrypt.png</normaloff>zuluCrypt.png</iconset>
  </property>
  <widget class="QWidget" name="centralWidget">
   <property name="contextMenuPolicy">
    <enum>Qt::ActionsContextMenu</enum>
   </property>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>641</width>
      <height>331</height>
     </rect>
    </property>
    <property name="contextMenuPolicy">
     <enum>Qt::ActionsContextMenu</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <property name="midLineWidth">
     <number>1</number>
    </property>
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="gridStyle">
     <enum>Qt::NoPen</enum>
    </property>
    <property name="rowCount">
     <number>0</number>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>Encrypted volume path</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Encrypted volume mount point path</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Type</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>659</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuOpen">
    <property name="title">
     <string>open</string>
    </property>
    <addaction name="actionFileOpen"/>
    <addaction name="actionPartitionOpen"/>
   </widget>
   <widget class="QMenu" name="menuCreate">
    <property name="title">
     <string>create</string>
    </property>
    <addaction name="actionFileCreate"/>
    <addaction name="actionPartitionCreate"/>
    <addaction name="actionCreatekeyFile"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>help</string>
    </property>
    <addaction name="actionInfo"/>
    <addaction name="actionAbout"/>
   </widget>
   <widget class="QMenu" name="menuConfigure">
    <property name="title">
     <string>luks</string>
    </property>
    <addaction name="actionAddKey"/>
    <addaction name="actionDeleteKey"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>options</string>
    </property>
    <addaction name="actionTray_icon"/>
    <addaction name="actionFonts"/>
   </widget>
   <widget class="QMenu" name="menuFavorites">
    <property name="title">
     <string>favorites</string>
    </property>
    <addaction name="actionManage_favorites"/>
   </widget>
   <addaction name="menuOpen"/>
   <addaction name="menuCreate"/>
   <addaction name="menuConfigure"/>
   <addaction name="menuFavorites"/>
   <addaction name="menuOptions"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionFileOpen">
   <property name="text">
    <string>encrypted file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionPartitionOpen">
   <property name="text">
    <string>encrypted partition</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionFileCreate">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>encrypted file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionPartitionCreate">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>encrypted partition</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>about</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+R</string>
   </property>
  </action>
  <action name="actionAddKey">
   <property name="text">
    <string>add key</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Q</string>
   </property>
  </action>
  <action name="actionDeleteKey">
   <property name="text">
    <string>delete key</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="actionInfo">
   <property name="text">
    <string>crypto info</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionCreatekeyFile">
   <property name="text">
    <string>key file</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+D</string>
   </property>
  </action>
  <action name="actionTray_icon">
   <property name="text">
    <string>tray icon</string>
   </property>
  </action>
  <action name="actionFonts">
   <property name="text">
    <string>select font</string>
   </property>
  </action>
  <action name="actionFavorite_volumes">
   <property name="text">
    <string>favorite volumes</string>
   </property>
  </action>
  <action name="actionManage_favorites">
   <property name="text">
    <string>manage favorites</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
